### ISSUE TITLE
Duda sobre opcion salir y volver

### ISSUE THREAD

**Issue by Nicromanker** [IIC2233/Syllabus#123]
### Prerrequisitos
* [X] Leí las reglas del foro (https://github.com/IIC2233/syllabus/issues/1)
* [X] Busqué en las *issues* si ya preguntaron mi duda y no encontré nada parecido (https://github.com/IIC2233/syllabus/issues)
* [X] Revisé el compilado de dudas de la tarea y no encontré una *issue* similar a la mía (https://github.com/IIC2233/syllabus/issues/111)
* [X] Mi duda no se trata sobre una librería, *built-in* o mala práctica, ya que eso se pregunta en la *issue* creada para ello.
* [X] Mi duda no se trata de un tema administrativo o personal, ya que en ese caso debo contactar a mi profe, al Jefe de Bienestar o al correo del curso (https://iic2233.github.io/contacto/)
* [X] Utilizaré un título descriptivo y llenaré correctamente esta plantilla
* [X] De ser necesario, solo colocaré código simple que permita explicar mi problema o duda, ya que compartir código de esta evaluación puede ser considerado copia

### Duda

Hola! Tengo una duda al dar al usuario la opcion de volver o salir. Una vez que el usuario elige la opcion de simular hora, tambien debe poder salir del juego desde ese "menu"? (sí permito que vuelva)

**Comment by 1862507J** [comment]
Sí. Desde todos los menús se debe poder "volver a atrás" y "salir".

**Comment by Nicromanker** [comment]
> Sí. Desde todos los menús se debe poder "volver a atrás" y "salir", por lo que en el menú de simulación de hora se debe poder salir también.

Entonces tambien desde el menu de eleccion de ataque?

**Comment by 1862507J** [comment]
Ya que el enunciado no dice lo contrario, también deberás poder "volver" y "salir" desde el menú de atacar a un tributo.

**Comment by emolaba** [comment]
Hola buenas! Al respecto de las opciones volver y salir, "salir" hace referencia a que termine el programa y "volver", por ejemplo, si transcurren x horas, sería ver el resumen de la hora anterior nuevamente o sería regresar al menú de inicio?
Muchas gracias de antemano!

**Comment by 1862507J** [comment]
Hola!
Sería regresar al menú principal.
Saludos
